internal/factsengine/factsengine_test.go

```diff
@@ -409,7 +408,7 @@ func (suite *FactsEngineTestSuite) TestFactsEngineLoadPlugins() {
 
 	engine := &FactsEngine{ // nolint
 		factGatherers: map[string]gatherers.FactGatherer{
-			gatherers.CorosyncFactKey: gatherers.NewCorosyncConfGatherer(),
+			gatherers.CorosyncCmapCtlFactKey: gatherers.NewCorosyncCmapctlGatherer(),
```

Comment:
Nothing wrong with it, but why this change?

Reply:
Basically because the CorosyncCmapctlGatherer doesn't require to pass any parameter to the constructor. But I guess that is changing soon enough :-)